autoware_lanelet2_map_validator/src/include/lanelet2_map_validator/validators/lane/border_sharing.hpp

```diff
@@ -42,11 +42,21 @@ class BorderSharingValidator : public lanelet::validation::MapValidator
 
 private:
   lanelet::validation::Issues check_border_sharing(const lanelet::LaneletMap & map);
+
+  /**
+   * @brief measure the length of entry/exit part, let middle = their average, and expand each point
+   * in left/right boundary by middle * (scale_factor-1)
+   */
```

Comment:
(It need new lines somewhere)

```suggestion
  /**
   * @brief return a polygon which is an expanded shape of the lanelet. each point will be expanded to the lateral direction, and its variation length is about (average length of both edges of the lanelet) * (scale_factor-1)
   */
```